recipes/mysql/post-link.sh

```diff
@@ -0,0 +1,64 @@
+#!/bin/bash
+
+# this script is based off the homebrew package:
+# https://github.com/Homebrew/homebrew-core/blob/master/Formula/mysql.rb
+# with lots of changes and additions to isolate database instances
+
+echo "MySQL Installation Notes:
+
+1. WARNING: MySQL has been installed without a root password. To secure the
+   installation, run:
+
+    mysql_secure_installation
+
+2. By default, MySQL will only allow connections from localhost.
+
+3. To start [stop] the server, run:
+
+    mysql.server start [stop]
+
+4. The MySQL data directory is located within your Conda environment. This
+   means that if your Conda environment is deleted, any databases will be
+   deleted too!" >> ${PREFIX}/.messages.txt
+
+# the default port is generated by hasing the conda env name
+if [ -x "$(command -v md5sum)" ]
```

Comment:
If you did this in python, with `hashlib`, you wouldn't have to do checks like this / it'd still work if whatever person's machine doesn't have either `md5` or `md5sum` on their `PATH`.

Reply:
Though actually, python isn't otherwise a runtime dep (and shouldn't be)...is there some simple in-bash way to do some version of this?

Reply:
We could use cksum which is the same on both OSX and linux and might be more portable.